src/main/java/com/faforever/moderatorclient/ui/MainController.java

```diff
@@ -69,6 +73,7 @@ public void initializeAfterLogin() {
         initVotingTab();
         initMessagesTab();
         initTutorialTab();
+		initReportTab();
```

Comment:
format?

Reply:
intellij was set to detect indents, all indents are spaces but it used tabs, hmm, interesting